pkg/operator/workloads/logs.go

```diff
@@ -441,20 +439,6 @@ func extractFromCortexLog(match string, loglevel string, logStr string) (*string
 		return formatHeader3(cutStr), false
 	}
 
-	matches := jsonPrefixRegex.FindStringSubmatch(cutStr)
```

Comment:
Was this removed to save space on `cx logs`?

Reply:
Yes, with the addition of printing the output of pre_inference and post_inference, a single prediction request was producing many log lines, even for something as small as iris